src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -295,6 +295,8 @@ TransactionManager::submit_transaction(
       SUBTRACET(seastore_t, "releasing projected_usage: {}", t, projected_usage);
       epm->release_projected_usage(projected_usage);
     });
+  }).finally([&t]() {
+    t.get_handle().exit();
```

Comment:
Ah, I see the IO was blocked at
```
0x617000060f00 trans.235 TransactionManager::submit_transaction: start, entering reserve_projected_usage
```
But I was instead wondering if it is caused by background cleaning. Looks to be a reasonable fix!